src/lib/y2security/security_policies/manager.rb

```diff
@@ -120,21 +159,39 @@ def failing_rules(config, scope: nil, include_disabled: false)
       # /etc/ssg-apply/default.conf file (if override.conf does not exist). Using an override.conf
       # file allows for custom configuration without modifying the default configuration file.
       #
-      # Only the #profile and #disabled-rules options are written.
-      def write_config
-        # Only one policy is expected to be enabled
-        policy = policies.find { |p| enabled_policy?(p) }
-
-        return unless policy
-
+      # Bear in mind that ssg-apply does not perform any kind of merging between both configuration
+      # files.
+      #
+      # Only the #profile and #remediate options are written.
+      def write_config(policy)
+        copy_default_config
```

Comment:
NP: could we have something like `file.save_as`? This would allow reading from one path, edit, and then save as another file.

Reply:
Yes, that was my first thought. But we need to fiddle with the internals of the `BaseModel` class (a rather fragile approach) or change that class. And, to be honest, I would rather prefer to limit these changes.